sandbox_others.go

```diff
@@ -0,0 +1,12 @@
+// +build !windows
```

Comment:
I think current convention is to name these files (shared by linux and freebsd) `_unix.go`